crew

```diff
@@ -449,6 +457,17 @@ def build_and_preconfigure (target_dir)
     end
     puts "Preconfiguring package..."
     @pkg.install
+
+
+    # Rename all *.la_tmp back to *.la
+    # find /usr/local/lib64 -type f -name '*.la_tmp' -exec sh -c 'mv "$1" "${1%.la_tmp}.la"' _  {} \;
+    puts "Rename all *.la_tmp files back to *.la".lightblue
+    if @opt_verbose then
+      system "find #{CREW_LIB_PREFIX} -type f -name '*.la_tmp' -exec sh -c 'mv \"$1\" \"${1%.la_tmp}.la\"' _  {} \\;"
+    else
+      system "find #{CREW_LIB_PREFIX} -type f -name '*.la_tmp' -exec sh -c 'mv \"$1\" \"${1%.la_tmp}.la\"' _  {} \\;"
+    end
+
```

Comment:
We have logic to delete la files for the procedure in which `make` , `make DESTDIR install` are performed. What I do is the procedure befor `make` to avoid libtool error.

Reply:
1. Rename *.la files in /usr/local/lib64
2. Configure / Compile code
3. Make install
4. Remove *.la files in DEST/usr/local/lib64 
5. Packages done

Reply:
Can you remove steps 4-5 if you delete the *.la files in step 1 instead?

Reply:
Why do we remove 4. 5.?

Reply:
We want to remove *.la files in new built package

Reply:
And we have problem to build new packages due to these la files.

Reply:
Step 4 makes sure no more .la files in new packages.
Step 5 make it possible to keep track of files list for old packges that have *.la files inside.

Reply:
I thought keeping *.la files hanging around causes run-time errors?

Reply:
why cuases run-time errors?

Reply:
*.la files are only archives.

Reply:
Maybe it was just build-time errors but the point is, why do we need them after the package is installed?

Reply:
crew files abc outputs something strange.

Reply:
`crew files abcde`?

Reply:
crew files "something"

Reply:
I don't know what you really concern about

Reply:
Not giving you the error below?
```
Package abc not found. :(
```

Reply:
I think the logic needs work.  We either need to delete the *.la files or leave them alone.  This renaming makes no sense.

Reply:
If you delete all *.´la files you get issues like  #2261.

Reply:
The reason you get those messages is because that package was compiled before my fix.  I do consider this an issue.

Reply:
With your fix #2264, compiling some packages suffers from  the problem "can not find /usr/local/lib/****.la issue"

Reply:
Try compiling that package from source and I bet the messages go away.

Reply:
Examples?

Reply:
#2315

Reply:
OK, I "think" I know what is going on here.  We need to delete the *.la files before the configure/make/make install process.  I'm not sure why some compiled packages were expecting *.la files and not finding them but that is one of the issues we need to solve also.